test/integration/patients/worklist/schedule.js

```diff
@@ -620,16 +620,28 @@ context('schedule page', function() {
       })
       .routeGroupsBootstrap(_.identity, testGroups)
       .routeDirectories(fx => {
-        fx.data = [{
-          attributes: {
-            name: 'Insurance Plans',
-            slug: 'insurance',
-            value: [
-              'BCBS PPO 100',
-              'Medicare',
-            ],
+        fx.data = [
+          {
+            attributes: {
+              name: 'Team',
+              slug: 'team',
+              value: [
+                'Coordinator',
+                'Nurse',
+              ],
+            },
```

Comment:
Added this filter. I use it to verify that the filters are alphabetically sorted:

1. Group
2. Insurance Plans
3. Team

If the filters aren't alphabetically sorted, `Team` would be positioned before `Insurance Plans`.